PULL_REQUEST_TEMPLATE

```diff
@@ -34,3 +32,4 @@ Provide instructions so we can reproduce.
 - [ ] I have added tests that prove my fix is effective or that my feature works
 - [ ] New and existing unit tests pass locally with my changes
 - [ ] Any dependent changes have been merged and published in downstream modules
+- [ ] Any UI changes have been checked to work on both desktop and mobile
```

Comment:
```
to work on different screen sizes
```
We should ensure that all screen sizes are working as expected (i.e tablet)?

Reply:
eugh i forgot about tablet 😭 

thanks